sdk/search/azure-search/src/main/java/com/azure/search/models/RangeFacetResult.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.models;
+
+/**
+ * Contains the range facet value found by a search query
+ */
+public class RangeFacetResult {
```

Comment:
This class should be marked as `@Immutable` given that it only has getters and all properties are `final`.